static/downloadables/connect_to_server.py

```diff
@@ -226,9 +286,14 @@ def formatGWASAndRetrieveClumps(GWASfile, GWASextension, GWASrefGen, refGen, sup
     # get clumps using the refGen and superpopulation
     clumpsData = getClumps(refGen, superPop, chromPos)
 
+    #get maf data using the refgen and mafcohort
+    fileName = "{m}_maf_{ahash}.txt".format(n=mafCohort, ahash=fileHash)
+    mafPath = os.path.join(workingFilesPath, fileName)
+    mafData = getMaf(mafCohort, refGen, chromPos)
+
     # get the study:snps info
     fileName = "traitStudyIDToSnps_{ahash}.txt".format(ahash = fileHash)
-    studySnpsPath = os.path.join(workingFilesPath, fileName)
+    studySnpsPath = os.path.join(workingFilesPath, fileName) #TODO do we need to do something for creating this?
```

Comment:
Will you handle this TODO in this push or in a later push?

Reply:
It is already done, just got to delete that comment